### PR TITLE
Feed style

### PR DESCRIPTION
**Pull request by Reine0017**
Styling for the reviews page (feed.html)

### CODE REVIEW

File: src/main/webapp/feed.html [hunk 1]
```diff
@@ -2,44 +2,71 @@
...
-          <option value="hi">Hindi</option>
-        </select>
-        <button onclick="submitTransReq();">Translate</button>
-        <select id="cinema">
-          <option value="all">All</option>
-        </select>
-        <button onclick="submitCinemaReq();">Select</button>
-        <div id="refreshInstruc">
-          <h2>Refresh page after 4.5s to see your review.</h2>
+    <img src="images/popcornIcon.png" alt="Image not available">
```
Comment (jiangyue12392) [inline src/main/webapp/feed.html]: Indent from line 14 to 23 is a bit wrong

File: src/main/webapp/feed.html [hunk 2]
```diff
@@ -2,44 +2,71 @@
...
+            <option value="all">All</option>
+          </select>
+          <button onclick="submitCinemaReq();">Select</button>
         </div>
-      </div>
-      <hr/>
-      <div id="message-container">Loading...</div>
+        <hr>
+        <div id="message-container">Loading...</div>
     </div>
```
Comment (jiangyue12392) [inline src/main/webapp/feed.html:72]: I think there might still be a `</div>` missing here

File: src/main/webapp/js/messages-loader.js [hunk 3]
```diff
@@ -86,9 +86,24 @@ function createCinemaOptions() {
     });
 }
 
+function loadOnce(){
+  if(!window.location.hash) {
+    window.location = window.location + '#loaded';
+    window.location.reload();
+  }
+}
+/*
```
Comment (jiangyue12392) [inline src/main/webapp/js/messages-loader.js]: Remove deadcode

File: src/main/webapp/js/messages-loader.js [hunk 4]
```diff
@@ -86,9 +86,24 @@ function createCinemaOptions() {
...
+  window.scroll(0, window.pageYOffset)
+};
+
+window.addEventListener("scroll", scrollEventHandler, false);*/
+
 // Fetch data and populate the UI of the page.
 function buildUI() {
   initializeUrlParams();
   createCinemaOptions();
+  setTimeout(loadOnce, 4500);
```
Comment (jiangyue12392) [inline src/main/webapp/js/messages-loader.js]: This seems to be from a separate PR

File: src/main/webapp/feed.html (hunk 1 above)
Comment (Reine0017) [reply src/main/webapp/feed.html]: fixed

File: src/main/webapp/feed.html (hunk 2 above)
Comment (Reine0017) [reply src/main/webapp/feed.html:72]: fixed

File: src/main/webapp/js/messages-loader.js (hunk 3 above)
Comment (Reine0017) [reply src/main/webapp/js/messages-loader.js]: removed

File: src/main/webapp/js/messages-loader.js (hunk 4 above)
Comment (Reine0017) [reply src/main/webapp/js/messages-loader.js]: removed

**Comment by Reine0017** [comment]
Can't resolve all conflicts right now because master branch has missing "div"

**Comment by jiangyue12392** [comment]
What do you mean cannot resolve all conflicts?

**Comment by Reine0017** [comment]
> What do you mean cannot resolve all conflicts?

@jiangyue12392 oh lol nvm, its all resolved now

**Review by jiangyue12392 — APPROVED** [review]
Looks good to me~
